src/app/ApplicationTemplate.Wasm/ApplicationTemplate.Wasm.csproj

```diff
@@ -33,9 +33,9 @@
     <PackageReference Include="Serilog.Sinks.Debug" Version="1.0.1" />
     <PackageReference Include="Uno.Microsoft.Toolkit.Uwp.DeveloperTools" Version="6.0.0-ge975abb046" />
     <PackageReference Include="Uno.UI.WebAssembly" Version="3.1.0-beta.84" />
-		<PackageReference Include="Uno.UI.RemoteControl" Version="2.4.0" />
+		<PackageReference Include="Uno.UI.RemoteControl" Version="3.6.6" />
 		<PackageReference Include="Uno.Injectable" Version="1.34.0-dev.175" />
-		<PackageReference Include="Uno.CodeGen" Version="1.32.0" />
+		<PackageReference Include="Uno.CodeGen" Version="1.34.0-dev.175" />
```

Comment:
-dev?

Reply:
well `Uno.CodeGen` and `Uno.Injectable` should be aligned.

Reply:
Should we go for dev or stable?